src/Data/Dao/PiantaDao.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+  namespace App\Data\Dao;
+  use App\Resources\Config\Database as Database;
+  use \PDO as PDO;
+  require_once __DIR__ . '/../../../resources/config/Database.php';
+
+  class PiantaDao
+  {
+
+    private $connection;
+    private $I_PLANT;
+    private $S_PLANT;
+    private $D_PLANT;
+    private $U_QR_PLANT;
+
+    public function __construct()
+    {
+      $Db = new Database();
+      $this->connection = $Db->connect();
+      $this->I_PIANTA = "INSERT INTO pianta(genere, specie, nome, co2, descrizione) VALUES(:genere, :specie, :nome, :co2, :descrizione)";
+      $this->D_PLANT = "DELETE FROM pianta WHERE id=:id";
+      $this->U_QR_PLANT = "UPDATE pianta SET qrcode = :qrcode WHERE id = :id";
+      $this->S_PLANT = "SELECT * FROM pianta AS a INNER JOIN stato_pianta AS b ON a.id = b.id_pianta WHERE a.id = :id ;";
+    }
+
+    public function store($Pianta)
+    {
+      $stmt = $this->connection->prepare( $this->I_PIANTA );
```

Comment:
@shewraar aggiungerei sempre un controllo ai metodi che prendono parametri in cui verifichi la correttezza del valore passato ... ad esempio che non sia `null` e cose di questo tipo

Reply:
ok